testplan/web_ui/testing/src/Report/BatchReport.js

```diff
@@ -114,6 +114,10 @@ class BatchReport extends React.Component {
             );
             axios.all([assertionsReq, structureReq]).then(
               axios.spread((assertionsRes, structureRes) => {
+                if (!assertionsRes.data) {
+                  console.error("The asserstion data is too large to parse!");
```

Comment:
I would suggest to make this error msg more generic - could there be other reason for assertions data to be empty?